src/nyc_trees/js/src/map.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+var L = require('leaflet');
+
+module.exports = {
+    create: create
+};
+
+function create() {
```

Comment:
Consider having the `create` function take the dom id of the map as an argument, to avoid hardcoding the dom element in a non-entry-point module (and in features coming very soon, to allow for more than one map on a page).